_dev/src/ts/components/ModalContainer.ts

```diff
@@ -0,0 +1,57 @@
+import Hydration from '../utils/Hydration';
+
+export default class ModalContainer {
+  public static readonly cancelEvent = 'cancel';
+  public static readonly okEvent = 'ok';
+
+  public static readonly containerId = 'ua_modal';
+
+  public init(): void {
```

Comment:
if you want to follow what you expected it's can be better to use `mount` and `beforeDestroy`  ?
```suggestion
  public mount(): void {
```